tools/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19
+FROM golang:1.20
```

Comment:
This breaks the linter, not sure why. I'll create a follow up PR to upgrade it to a version that supports 1.20, can you revert this change in the meantime?

Reply:
Hi, sure.